frontend/app/api/movies/[id]/reviews/route.ts

```diff
@@ -7,13 +7,18 @@ interface Params {
 export async function GET(req: NextRequest, { params }: { params: Params }) {
   try {
     const movieId = parseInt(params.id);
+    if (isNaN(movieId)) {
+      return NextResponse.json({ error: "Movie id not a number" }, { status: 400 });
+    }
+    if (movieId <= 1) {
+      return NextResponse.json({ error: "Movie id cant be under 1" }, { status: 400 });
+    }
     const result = await getMovieReviewsById(movieId);
     if (result.length === 0) {
       return NextResponse.json({ error: `Movie not found with id ${movieId}` }, { status: 404 });
```

Comment:
Korjaus: Reviews not found for movie id ${movieId}

Reply:
Okei Tätä iffiä ei tarvii. Parempi olisi vain palauttaa tyhjä array. Tarvitaan sitten vähemmän käsittelyä frontin puolella